charts/consul/values.yaml

```diff
@@ -436,6 +436,33 @@ global:
       # @type: string
       secretKey: null
 
+    # The resource requests (CPU, memory, etc.) for the server-acl-init and server-acl-init-cleanup pods. 
+    # This should be a YAML map corresponding to a Kubernetes
+    # [`ResourceRequirements``](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#resourcerequirements-v1-core)
+    # object.
+    #
+    # Example:
+    #
+    # ```yaml
+    # resources:
+    #   requests:
+    #     memory: '200Mi'
+    #     cpu: '100m'
+    #   limits:
+    #     memory: '200Mi'
+    #     cpu: '100m'
+    # ```
+    #
+    # @recurse: false
+    # @type: map
+    resources:
+      requests:
+        memory: "50Mi"
+        cpu: "50m"
+      limits:
+        memory: "50Mi"
+        cpu: "50m"
```

Comment:
Are these what the values defaulted to in the absence of us providing them? Just wanted to know if we might be concerned this is not enough, but otherwise ignore this!!

Reply:
Yes, these were the previous hard-coded values. As to whether they are enough, the practitioner reported that they get OOM because they have a large number (50+) of secondary clusters. Since it's just a job, we could double everything, and I don't know if most folks would mind, but this is also an extreme use case. I would defer to you if you have a strong opinion. WDYT @thisisnotashwin ?

Reply:
nah! this rationale works for me! thank you for all this information!! i really appreciate it!!